src/stm32/stm32_spi_master_gspi.c

```diff
@@ -128,7 +129,9 @@ bool stm32_gspi_run_txn_fd(struct mgos_spi *c, const struct mgos_spi_txn *txn) {
       if (c->debug) {
         LOG(LL_DEBUG, ("read 0x%02x", byte));
       }
-      *rx_data++ = byte;
+      if(!discard_rx) {
```

Comment:
space after if

Reply:
I failed to use the clang auto-formatter because it changed half the file... our code style is no spaces so my automatic action is not to type them. I'll run a selection format this time

Reply:
i checked in and pushed our standard .clang-format to this repo, you should be able to use clang-format now

Reply:
https://github.com/mongoose-os-libs/spi/blob/master/.clang-format

Reply:
Great! Done